go.mod

```diff
@@ -281,8 +281,6 @@ replace (
 	github.com/osmosis-labs/osmosis/x/epochs => github.com/osmosis-labs/osmosis/x/epochs v0.0.5-0.20240825083448-87db4447a1ff
 	github.com/osmosis-labs/osmosis/x/ibc-hooks => github.com/osmosis-labs/osmosis/x/ibc-hooks v0.0.0-20240825083448-87db4447a1ff
 
-	github.com/osmosis-labs/sqs/sqsdomain => ./sqsdomain
```

Comment:
Wasn't this one was intended to point to a local copy instead of remote for development so we're on `v26` branch local version, path should be correct anyways?